README.md

```diff
@@ -351,3 +351,25 @@ You can create your on commands by using the lua functions this plugin provides
 command! -count=1 TermGitPush  lua require'toggleterm'.exec("git push",    <count>, 12)
 command! -count=1 TermGitPushF lua require'toggleterm'.exec("git push -f", <count>, 12)
 ```
+
+### Open multiple terminals side-by-side
+| Direction | Tested |
+| --- | --- |
+| vertical | ✔️ |
+| horizontal | ✔️ |
+| window | ⚠️ |
```

Comment:
These are unsupported by definition e.g. a window is supposed to occupy the whole screen so by definition you can show multiple window mode terms side by side, also this is unsupported for floats I would just put an `X` there to explain that it isn't supported